### DISCUSSION TITLE
useDisclosure in next js

### DISCUSSION THREAD

**Discussion by bisarroyo** [heroui-inc/heroui#1375] [Help]
I´m trying to use modal in next js but i got this error:

The error was caused by importing '@nextui-org/use-disclosure/dist/index.mjs'

I´m importing using:
import { useDisclosure } from '@nextui-org/use-disclosure'

How can I fix this error?

**Comment by Adamkaram** [comment]
did u put 'use client' in top of ur component ?

**Reply by bisarroyo** [reply]
I added 'use client' but does not work

**Comment by jrgarciadev** [comment]
Did you install the package `@nextui-org/use-disclosure` ? you can also import it from the main package `@nextui-org/react`

**Comment by pawitpr** [comment]
import it from import { useDisclosure } from '@nextui-org/modal';,

**Comment by pawitpr** [comment]
or directly use from @next-ui/react
